lib/features/thread/presentation/thread_controller.dart

```diff
@@ -188,12 +188,13 @@ class ThreadController extends BaseController with EmailActionController {
           _openEmailWithoutMailboxFromLocationBar(success.email);
         }
       }
+    } else {
+      super.handleSuccessViewState(success);
     }
   }
 
   @override
   void handleFailureViewState(Failure failure) {
-    super.handleFailureViewState(failure);
```

Comment:
why we need to change order of this method. I look risky for the whole thread controller

Reply:
To keep the semantic logic of `if/else` consistent, avoid misunderstandings about the parent function of this function always being called first. We only execute the parent function when the child function's conditions are not satisfied. This change does not affect the logic of the current threads.

Reply:
reverted